### ISSUE TITLE
Changes in version 2.0.0?

### ISSUE THREAD

**Issue by jsfeldman** [aspnet/JavaScriptServices#1295]
I'm trying to upgrade all my .NET packages to 2.0.0, however, I'm having difficulty running my development webpack configuration using SpaServices/WebpackDevMiddlewareOptions with the 2.0.0 package. I've updated aspnet-webpack to 2.0.1. Are there any other changes I need to make in order to upgrade to 2.0.0? I'm just using the vue-cli webpack template [configuration](https://github.com/vuejs-templates/webpack/tree/develop/template/build), so I don't think I'm doing anything too unique.

It seems to complain about webpackJsonp being undefined, but since my development configuration doesn't even use the CommonsChunkPlugin (as mentioned [here](https://github.com/webpack/webpack/issues/368)) and the code also appears minified even though it's not configured to minify in development I believe my code might be building wrong. I thought it might be building using production's configuration, but I commented out that configuration file and it is still building the same way.

I tried looking into possible causes from other projects but I believe I narrowed it down to upgrading the package (it seems to run fine by just calling webpack-dev-server directly from the command line). 

Has anyone else experienced this issue? Let me know if there is any other information I could give you that would be helpful or if I should ask anywhere else.

**Comment by SteveSandersonMS** [comment]
If you want to update your packages, I'd recommend checking which combination of packages is used by newly-created 2.0 SPA apps, and ensure you have the same set of references. I think `aspnet-webpack` 2.0+ requires Webpack 2.0+, but you should check and get a consistent set of references like newly-created projects have. Hope that's OK!

**Comment by jsfeldman** [comment]
Hello Steve,

Thanks for answering my very vague question haha. I looked into it more and I believe I've narrowed down the cause. I think the issue I'm experiencing might have something to do with my output configuration. Mine is configured like so: 

```
module.exports = {
  entry: {
    app: './src/main.js'
  },
  output: {
    path: path.resolve(__dirname, '../dist'),
    publicPath: '/',
    filename: 'js/[name].js'
  },
  ...
}
```

But I noticed that all the examples place the output path in wwwroot like this:

```
module.exports = {
  entry: {
    app: './src/main.js'
  },
  output: {
    path: path.resolve(__dirname, '../../wwwroot/dist'), // My ProjectPath is a sub directory of my c# project
    publicPath: '/dist/',
    filename: 'js/[name].js'
  },
  ...
}
```

When I switched my configuration to the above everything started working (except webpack didn't serve my index.html file when navigating to the root, only at /dist/index.html, but I think that may be due to other configuration). I even commented out  `app.UseStaticFiles()` to, hopefully, ensure it was the WebpackDevMiddleware serving the files and not the StaticFiles middleware.

Was some change made in 2.0.0 that required the webpack output to be at wwwroot? I noticed that I don't experience the issue in version 2.0.0-preview2-final and am able to use the second configuration without any problems. I tried changing the output path a few other ways, but putting it in wwwroot was the only one that worked for me.
